features/step_definitions/email_inbox_home_page.rb

```diff
@@ -0,0 +1,8 @@
+Given("I am logged into my account") do
+expect "Signed in successfully."
```

Comment:
Why is this file in the PR?

Reply:
should've been deleted. is gone now, sorry.